src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,27 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  width: $calendarWidth;
+  padding-top: 50px;
+
+  &__days {
+    width: 98px;
+    height: 98px;
+    border: 1px solid #000;
+    background-color: #eee;
```

Comment:
i guess would be better to add colors in variables

Reply:
i did not add because i used them just 1 time